src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -149,9 +166,12 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
       stay()
     case Event(ExecutionStoreCreated(startMode), NoFailureMessage) =>
       if (startMode == Restart) {
-        symbolsMarkdownTable foreach { table => log.info(s"Initial symbols:\n\n$table") }
-        executionsMarkdownTable foreach { table => log.info(s"Initial executions:\n\n$table") } 
+        dumpTables()
+        stay()
+      } else {
+        startRunnableCalls()
       }
+    case Event(TablesDumped, NoFailureMessage) =>
       startRunnableCalls()
```

Comment:
OK I can try to rework this.  Just to manage expectations, there likely will end up being more events as a consequence of `Await.result` removal since we can't asynchronously transition states in the FSM.  But that doesn't mean I need to do it the way it is now.  :smile: